ruby-challenges.rb

```diff
@@ -10,22 +10,68 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def letter_array (array, letter)
+    array.select {|value| value.include?(letter)}
+    end
+
+
+ letter_array(beverages_array, letter_o)
+ letter_array(beverages_array, letter_t)
+
```

Comment:
How are you seeing the output? Please make use of the terminal to printout the output.